### PR TITLE
[OneExplorer] Add OpenContainingFolder Command

### PR DESCRIPTION
**Pull request by dayo09**
This commit adds open containing folder command.

ONE-vscode-DCO-1.0-Signed-off-by: Dayoung Lee <dayoung.lee@samsung.com>

### CODE REVIEW

**Comment by dayo09** [comment]
### GIF

[link](https://github.sec.samsung.net/RS7-RuntimeNTools/secret/issues/187#issuecomment-1527227)

**Review by llFreetimell — APPROVED** [review]
LGTM

File: src/OneExplorer/OneExplorer.ts [hunk 1]
```diff
@@ -165,6 +165,10 @@ export class OneTreeDataProvider implements vscode.TreeDataProvider<OneNode> {
 
   // TODO: Add move()
 
+  openContainingFolder(oneNode: OneNode): void {
+    vscode.commands.executeCommand('revealFileInOS', oneNode.node.uri);
+  }
+
```
Comment (yunjayh) [inline src/OneExplorer/OneExplorer.ts:171]: Honestly, I expected the `open containing folder` button will show the folder in file explorer view, which can be executed by ` vscode.commands.executeCommand('revealInExplorer', oneNode.node.uri);`.
I'm really sorry if it has been discussed before, but can I get the background of opening the os' file explorer?

Comment (dayo09) [reply src/OneExplorer/OneExplorer.ts:171]: I think those commands means to do the same, based on their name..? Where is the command id from? I tried to grep one from vscode api docs, and I cannot find one. 

I can find the revealFileInOs [here](https://code.visualstudio.com/updates/v1_10)

Comment (yunjayh) [reply src/OneExplorer/OneExplorer.ts:171]: After our offline talk, I got the history about this feature (popping up OS File Window). It's requested in our discussion, and the feature what I expected is already available by clicking *file explorer icon*. :smile:

**Review by yunjayh — APPROVED** [review]
LGTM Thank you!

**Pull request merged**